site/content/en/blog/releases/1.27.0.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "1.27.0 - Go 1.19 and Cluster Autoscaler Improvements"
+linkTitle: "1.27.0"
+date: "2022-10-25"
+---
+# v1.27.0
```

Comment:
```suggestion
```

Drop this line please, since it's in the `title` we don't need it. (I know it's in the template, it's only there as a reminder to make the `title).